templates/repo/settings/webhook/settings.tmpl

```diff
@@ -237,6 +237,22 @@
 	<span class="help">{{.locale.Tr "repo.settings.branch_filter_desc" | Str2html}}</span>
 </div>
 
+<!-- Authorization Header -->
+{{if eq .HookType "matrix"}}
+<!-- add the "required" class and attribute, as well as a placeholder="Bearer $access_token", like in https://spec.matrix.org/v1.3/client-server-api/#client-authentication -->
+<div class="required field">
+	<label for="authorization_header">{{.locale.Tr "repo.settings.authorization_header"}}</label>
+	<input id="authorization_header" name="authorization_header" type="text" value="{{ .Webhook.HeaderAuthorization }}" placeholder="Bearer $access_token" required>
+	<span class="help">{{.locale.Tr "repo.settings.authorization_header_desc" | Str2html}}</span>
+</div>
+{{ else }}
+<div class="field">
+	<label for="authorization_header">{{.locale.Tr "repo.settings.authorization_header"}}</label>
+	<input id="authorization_header" name="authorization_header" type="text" value="{{ .Webhook.HeaderAuthorization }}">
+	<span class="help">{{.locale.Tr "repo.settings.authorization_header_desc" | Str2html}}</span>
+</div>
+{{ end }}
```

Comment:
I think this should be reduced to one `field`, with two inline `if` for the differences.

Reply:
Ok, I updated this part :+1: